components/form/demo/form-panel-demo.js

```diff
@@ -32,28 +32,9 @@ class FormPanelDemo extends LitElement {
 				display: none;
 			}
 
-			.d2l-form-panel-demo-panel {
-				background-color: white;
-				border-radius: 8px;
-				padding: 20px;
-			}
-
 			.d2l-form-panel-demo-container {
 				margin-bottom: 10px;
 			}
-
-			.d2l-form-panel-demo-header {
-				align-items: top;
```

Comment:
`top` is not a supported value for `align-items`. This css was also just no longer used so I cleaned it up.